BUILDING.md

```diff
@@ -115,6 +115,8 @@ Afterwards:
 
 `Build > Build Solution`
 
+If you're receiving errors regarding `constinit` then you need to change the language standard in the project's properties.
```

Comment:
This comment is kinda pointless, since the only reason why the language standard would be wrong is that you meddled with it in the first place.

Reply:
Sorry, forgot to remove that part before committing.